tests/nnstreamer_source/unittest_src_iio.cc

```diff
@@ -1373,8 +1371,6 @@ TEST (testTensorSrcIio, dataVerifyCustomChannels)
   EXPECT_EQ (config.info.info[0].type, _NNS_FLOAT32);
   EXPECT_EQ (config.info.info[0].dimension[0], 2U);
   EXPECT_EQ (config.info.info[0].dimension[1], 1U);
-  EXPECT_EQ (config.info.info[0].dimension[2], 1U);
-  EXPECT_EQ (config.info.info[0].dimension[3], 1U);
```

Comment:
EXPECT_LT ?

Reply:
IMO it is better checking 0 dim clearly here. please review the changes again.